pkg/cmd/miscellaneous.go

```diff
@@ -350,17 +361,84 @@ func getRole() string {
 }
 
 /*
-getTechnicalID returns the Technical Id, which used as clustername of the shoot cluster
+getTargetMapInfo retun garden,project,seed,shoot,TechnicalID to global getTargetInfo
 */
-func getTechnicalID() string {
+func getTargetMapInfo() map[string]string {
```

Comment:
the return value of this function is never used currently.

As the target stack can't change within one execution of `gardenctl` you could early-exit the function in case you have already initialized your targetMapInfo.

Reply:
I am add one global variable `var getTargetInfo map[string]string = make(map[string]string)` in `root.go`

once getTargetMapInfo() call in go file, it will return map value to global variable `getTargetInfo` which keep full information (garden,seed,shoot,shootTechnicalID).  then we can use  `getTargetInfo[key]=value` in multiple place in one go file without call   getTargetMapInfo() to query all the information again.  that the logic I set up so far.

Reply:
oh missing this part will add a check map variable whether empty tomorrow. 

> As the target stack can't change within one execution of `gardenctl` you could early-exit the function in case you have already initialized your targetMapInfo.

Reply:
done